nss-cohort-36/bangazon-client-imagination-station-react
# Ts ticket 17 cust profile details

# Description

Can edit profile information

## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. fetch and checkout
1. After merging API PR, go to profile, select edit info, change info, and you should see updated profile info.
# Checklist:
